openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/WorkflowRepository.java

```diff
@@ -14,8 +14,8 @@
 
 public class WorkflowRepository extends EntityRepository<Workflow> {
 
-  private static final String UPDATE_FIELDS = "owner";
-  private static final String PATCH_FIELDS = "owner";
+  private static final String UPDATE_FIELDS = "owner,status,response";
```

Comment:
If we don't want to update `status` and `response` in a PUT operation, we can remove those.

```suggestion
  private static final String UPDATE_FIELDS = "owner";
```